app/models/condition_set.rb

```diff
@@ -33,6 +33,10 @@ def approved_conditions
     conditions.joins(:validation_requests).where(validation_requests: {approved: true})
   end
 
+  def not_cancelled_conditions
+    conditions.reject(&:cancelled_at)
```

Comment:
I'm not sure if it matters but is it better to use the active record method for this?

```suggestion
    conditions.where(cancelled_at: nil)
```

I _think_ `reject` might select all of them and then filter, instead of filtering in the database.